code/modules/mob/living/carbon/human/species/station.dm

```diff
@@ -452,6 +452,9 @@
 	blood_color = "#1F181F"
 	flesh_color = "#AAAAAA"
 	reagent_tag = PROCESS_SYN
+	
+	has_organ = list(
+		)
```

Comment:
The ideal solution would be to port Bay's IPC organs. Right now we could give them a mechnical heart and mechanical eyes. I don't think posibrains currently work with IPC's, or at least they don't spawn them right now.

For now this is better than them spawning appendices upon gibbing.